FetchMigration/python/migration_monitor.py

```diff
@@ -97,7 +98,7 @@ def run(args: argparse.Namespace, wait_seconds: int) -> None:
         type=int,
         help="Target doc_count to reach, after which the Data Prepper pipeline will be terminated"
     )
-    cli_args = arg_parser.parse_args()
+    namespace = arg_parser.parse_args()
```

Comment:
nit: namespace threw me off a little, was expecting something like monitor_input, same thing for pre_migration main